autoload/ale/lsp_linter.vim

```diff
@@ -130,6 +130,11 @@ function! ale#lsp_linter#HandleLSPResponse(conn_id, response) abort
         call s:HandleLSPErrorMessage(l:linter_name, a:response)
     elseif l:method is# 'textDocument/publishDiagnostics'
         call s:HandleLSPDiagnostics(a:conn_id, a:response)
+    elseif l:method is# 'window/showMessage'
+        call ale#lsp#window#showMessage(
+        \ s:lsp_linter_map[a:conn_id],
+        \ g:ale_lsp_show_message_format,
+        \ a:response.params)
```

Comment:
```suggestion
        \ a:response.params
      )
```

Put the closing parenthesis on its own line, and use 4 space indenting for Vim source files.